.gitlab-ci.yml

```diff
@@ -63,30 +38,47 @@ before_script:
   CI_PLATFORM: "do"
   SSH_USER: root
 
+.packet_variables: &packet_variables
+  CI_PLATFORM: "packet"
+  SSH_USER: kubespray
 
-.testcases: &testcases
-  <<: *job
-  <<: *docker_service
+.packet-job: &packet-job
+  tags:
+    - packet
+  image: quay.io/kubespray/kubespray:v2.8.1
+
+.gce-job: &gce-job
+  tags:
+    - kubernetes
+    - docker
+  image: quay.io/kubespray/kubespray:v2.8.1
+
+.docker_service: &docker_service
+  services:
+     - docker:dind
+
+.cache: &cache
   cache:
     key: "$CI_BUILD_REF_NAME"
     paths:
       - downloads/
       - $HOME/.cache
+
+.testcases: &testcases
   before_script:
-    - docker info
-    - /usr/bin/python -m pip install -r requirements.txt
+    #- /usr/bin/python -m pip install -r requirements.txt
```

Comment:
should keept it

Reply:
it is not used in any ci job, so need to.